task/rhacs-deployment-check/3.71/rhacs-deployment-check.yaml

```diff
@@ -64,20 +81,15 @@ spec:
           value: $(params.insecure-skip-tls-verify)
         - name: OUTPUT
           value: $(params.output_format)
-        - name: ROX_API_TOKEN
-          valueFrom:
-            secretKeyRef:
-              name: $(params.rox_api_token)
-              key: rox_api_token
-        - name: ROX_CENTRAL_ENDPOINT
-          valueFrom:
-            secretKeyRef:
-              name: $(params.rox_central_endpoint)
-              key: rox_central_endpoint
+        - name: ROX_CONFIG_DIR
+          value: $(params.rox_config_dir)
+        - name: ROX_API_TOKEN_FILE
+          value: $(params.rox_token_file)
```

Comment:
Wondering whether using `$(workspaces.rox-api-token-auth.path)/$(params.private-key-path)` would be better, this way folks only have to specify the path _within_ the workspace without the hardcoded prefix. However, I'm not entirely sure what will happen if the workspace isn't there since it's optional; but this would ease configuration (and looking at some other tasks in the catalog; also what seems to be done most of the time).

Reply:
This would require roxctl to be able to check if the file can be opened before using it as token source. This can still be changed.
In that case both `rox_token_file` and `rox_config_dir` should be relative paths in the associated workspaces.